autowarefoundation/autoware
# Feature/ndt pcl gpu

## Status
Development

## Description
Added ndt_cpu library

YamatoAndo wrote:
Can be executed with the same implementation as the NDT library implemented for the GPU on the CPU (it is implemented differently from the original PCL)

## Todos


## Steps to Test or Reproduce
Check use_fast_pcl in app dialogue

YamatoAndo wrote:
Since the execution result is almost the same as the ordinary implementation,
It is OK if you can confirm that it moves as usual.
